pipelines/tests/test_pipelines.py

```diff
@@ -1,29 +1,15 @@
-import unittest
 import os
-import sys
-
-from dwave.system.samplers import DWaveSampler
-
-try:
-    DWaveSampler()
-    _config_found = True
-except ValueError:
-    _config_found = False
+import subprocess
+import unittest
 
-# /tests/integration/test_demo.py
-PY2 = sys.version_info.major == 2
+# /tests/test_demo.py
```

Comment:
Did you mean something like `/path/to/demos/pipelines/tests/test_pipelines.py` here?